### ISSUE TITLE
Bump version for bitgo dependency

### ISSUE THREAD

**Issue by rudygodoy** [BitGo/bitgod#9]
After a clean install, either using `yarn` or `npm`; the software doesn't work because their dependency for `bitgo` is also `4.3.1`, which doesn't explicitly state[1] `bluebird` as dependency since it uses it[2] in `src/v2/baseCoin.js`

1- https://github.com/BitGo/BitGoJS/blob/4.3.1/package.json#L69
2- https://github.com/BitGo/BitGoJS/blob/4.3.1/src/v2/baseCoin.js#L9

```
--- ~ » bitgod -h
module.js:471
    throw err;
    ^

Error: Cannot find module 'bluebird'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/test/.npm-global/lib/node_modules/bitgod/node_modules/bitgo/src/v2/baseCoin.js:9:17)
```

**Comment by rudygodoy** [comment]
There is already a commit for this: 0ec1b5f5f987186a6aac4f075d867b6bf92759c1

**Comment by Xekyo** [comment]
Thanks for the report, we recently released 4.18.1. I would expect that this issue is resolved.
